taskcluster/docker/android-build/Dockerfile

```diff
@@ -7,13 +7,18 @@ FROM $DOCKER_IMAGE_PARENT
 
 VOLUME /builds/worker/checkouts
 
+# Install Java 8 needed for Nexus
+RUN apt-get update -qq \
+    && apt-get install -y openjdk-8-jdk \
+    && apt-get clean
```

Comment:
Oh noes! The Android SDK license is not accepted anymore[1]:


```
[task 2023-03-07T17:14:00.704Z] > Failed to install the following Android SDK packages as some licences have not been accepted.
[task 2023-03-07T17:14:00.704Z]      platforms;android-33 Android SDK Platform 33
[task 2023-03-07T17:14:00.704Z]      build-tools;30.0.3 Android SDK Build-Tools 30.0.3
```

I wonder if the license is tied to the version of Java we're running?

[1] https://firefox-ci-tc.services.mozilla.com/tasks/Uio-go4KQ9euS6RJfgOnrA/runs/5/logs/public/logs/live.log

Reply:
@JohanLorenzo how is the license currently being accepted? For Focus, this happened here, but I don't see where we do this for external-gradle-dependencies: https://github.com/mozilla-mobile/firefox-android/pull/1077/files#diff-d33eee172ac4faa57c5063f0cbfbf224d0ff79d7d7dc6e917bbd3e12ebfd0df5R49

Reply:
OK, the next missing piece was switching to `cmdline-tools/latest/bin/sdkmanager` which supports Java 11, as opposed to `tools/bin/sdkmanager` which only supports Java 8. Let's see what's next :)

Reply:
Next we need to download a newer version of the Android SDK tools in our fetch task https://github.com/mozilla-mobile/firefox-android/blob/c91fe073d1984ae2ba6ceee500a72309dd25e11c/taskcluster/ci/fetch/kind.yml#L20.

Reply:
@JohanLorenzo Alright, I've got this to work now! We're using Java 11 now, except for Nexus which doesn't support it, but it does also upgrade Nexus to the latest version. The previous one was from 2016, so I guess we should keep the upgrade.